pgjdbc/src/test/java/org/postgresql/test/jdbc4/ArrayTest.java

```diff
@@ -560,7 +560,7 @@ public void testToString() throws SQLException {
   public void nullArray() throws SQLException {
     PreparedStatement ps = con.prepareStatement("INSERT INTO arrtest(floatarr) VALUES (?)");
 
-    ps.setNull(1, Types.ARRAY, "float8");
+    ps.setNull(1, Types.ARRAY, "float8[]");
```

Comment:
This is an interesting side effect. I think it is worth including in a notable changes. So clients get prepared for "unexpected" operation of setNull (since String argument has been just ignored for ages)